### DISCUSSION TITLE
What is the next plan to progress this project?

### DISCUSSION THREAD

**Discussion by prokawsar** [ally-ui/ally-ui#56] [Q&A] [answered]
Hello
I have been visiting codes and website, it looks amazing, but didn't find any next plan or progress checklist.
I'm working as Frontend Dev in Svelte, So would like to know how could I contribute to this project for Svelte components and what is the process?
Also would like to know the next progressive plan

Thanks
Kawsar

**Comment by bryanmylee** [comment, marked as answer]
Hi @prokawsar, thanks for your interest in the project!

I'm currently working on improvements to the core state system, but once that is done I will be polishing up the dialog component's focus trap system.

Then, we would want to tackle some low hanging fruit with easy components such as the switch and checkboxes.

This project is still in alpha and a lot of things are not confirmed yet, but if you'd like to join me in building it, I'd love to have your help and support :)

**Reply by bryanmylee** [reply]
@prokawsar You can reach out for more details on the Discord :)

**Reply by prokawsar** [reply]
yeah, sure. Thanks a lot for reply.
